rust/release/src/os_arch.rs

```diff
@@ -127,6 +127,24 @@ impl OsArch {
         Arch::current() == arch
     }
 
+    /// As `ax` was introduced, the `linux-arm` architecture was dropped.
+    /// This method is in every way the same as `all`,
+    /// except it does not return said architecture.
```

Comment:
This description makes sense (don’t yet know why we still need `linux-arm` in `all`), so I’d have the implementation filter the result of `Self::all()`.

Reply:
Maybe I can remove it, to be honest, I just did the smallest possible change that _definitely_ doesn't break anything.